dcos/marathon.py

```diff
@@ -741,10 +743,30 @@ def remove_pod(self, pod_id, force=False):
         :rtype: None
         """
 
-        path = 'v2/pods' + _normalize_marathon_id_path(pod_id)
+        path = self._marathon_id_path_join('v2/pods', pod_id)
         params = {'force': 'true'} if force else None
         self._rpc.http_req(http.delete, path, params=params)
 
+    @staticmethod
+    def _marathon_id_path_join(url_path, id_path):
+        """Concatenates a URL path with a Marathon "ID path", ensuring the
+        result is well-formed.
+
+        The path and the ID will be joined with a single forward slash (/),
+        all trailing slashes in the ID will be removed, and the ID will have
```

Comment:
Could you give an example? Looking at the existing code I don't see anything ensuring trailing slashes are present. Did you mean a _leading_ slash, i.e. we want `/foo/bar` instead of `foo/bar`?